src/main/java/bio/terra/app/configuration/PolicyServiceConfiguration.java

```diff
@@ -6,36 +6,15 @@
 import java.io.IOException;
 import java.util.List;
 import org.springframework.boot.context.properties.ConfigurationProperties;
-import org.springframework.boot.context.properties.EnableConfigurationProperties;
-import org.springframework.context.annotation.Configuration;
+import org.springframework.boot.context.properties.ConstructorBinding;
 
 /** Configuration for managing connection to Terra Policy Service. * */
-@Configuration
-@EnableConfigurationProperties
 @ConfigurationProperties(prefix = "tps")
-public class PolicyServiceConfiguration {
-
-  private boolean enabled;
-  private String basePath;
+@ConstructorBinding
+public record PolicyServiceConfiguration(boolean enabled, String basePath) {
 
   private static final List<String> POLICY_SCOPES = List.of("openid", "email", "profile");
 
-  public boolean getEnabled() {
-    return enabled;
-  }
-
-  public void setEnabled(boolean enabled) {
-    this.enabled = enabled;
-  }
-
-  public String getBasePath() {
-    return basePath;
-  }
-
-  public void setBasePath(String basePath) {
-    this.basePath = basePath;
-  }
-
   public String getAccessToken() throws IOException {
     GoogleCredentials credentials =
         ServiceAccountCredentials.getApplicationDefault().createScoped(POLICY_SCOPES);
```

Comment:
It's a minor thing, but I think `getApplicationDefault()` should use the class that it's declared in for scoping rather than a subclass
```suggestion
        GoogleCredentials.getApplicationDefault().createScoped(POLICY_SCOPES);
```